docs/reference/indices/flush.asciidoc

```diff
@@ -39,7 +40,153 @@ or even on `_all` the indices.
 
 [source,js]
 --------------------------------------------------
-$ curl -XPOST 'http://localhost:9200/kimchy,elasticsearch/_flush'
+POST /kimchy,elasticsearch/_flush
 
-$ curl -XPOST 'http://localhost:9200/_flush'
+POST /_flush
 --------------------------------------------------
+// AUTOSENSE
+
+[[indices-synced-flush]]
+=== Synced Flush
+
+Elasticsearch tracks the indexing activity of each shards. Shards that have not
+received any indexing operations for 30 minutes (configurable) are automatically marked as inactive. This presents
```

Comment:
Yeah - probably a good call. Just making it a link to an anchor tag would be great!